### DISCUSSION TITLE
sript blocked fix

### DISCUSSION THREAD

**Discussion by monkeynosee** [LeDragoX/Win-Debloat-Tools#81] [Ideas]
script blocked fix First run in powershell  Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser

**Comment by LeDragoX** [comment]
This is explained at <https://github.com/LeDragoX/Win-Debloat-Tools#-usage-requirements>

I don't know if it was hard to understood, but i'll rewrite that part.
